civitai-downloader-v2/src/data/schema_manager.py

```
@@ -0,0 +1,375 @@
+        try:
+            conn = sqlite3.connect(
+                str(db_path),
+                timeout=30.0,
+                check_same_thread=False
+            )
+            # Enable foreign keys
+            conn.execute("PRAGMA foreign_keys = ON")
+            # Set row factory for dict-like access
+            conn.row_factory = sqlite3.Row
+            yield conn
+        except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)